### ISSUE TITLE
Save changes modal is not screen reader accessible.

### ISSUE THREAD

**Issue by dynamic11** [CDH-Studio/UpSkill#1475]
**Describe the bug**
the screen reader does not read the content of the modal

**To Reproduce**

1. Go to any edit profile form
2. Make an edit
3. clock on any other form without saving
4. See the modal that pops up

**Expected behavior**
The screen reader should read the content of every modal. If we use another type of antd modal it should fix this issue

**Screenshots**
![image](https://user-images.githubusercontent.com/11470442/122436783-18d31580-cf67-11eb-97f7-543b53d8fbb3.png)

**Additional context**
